lib/phoenix/template.ex

```diff
@@ -199,14 +222,14 @@ defmodule Phoenix.Template do
       Returns the template root alongside all templates.
       """
       def __templates__ do
-        {@template_root, unquote(names)}
+        {@template_root, @lookup_pattern, unquote(names)}
```

Comment:
I don't think we need to return the pattern here because it doesn't seem like nothing uses it?